crawl-ref/source/player.h

```diff
@@ -200,6 +200,7 @@ class player : public actor
     int lives;
     int deaths;
 #if TAG_MAJOR_VERSION == 34
+    // Need to keep these fields for save compat reasons.
```

Comment:
I don't think we do. You can remove them from player.h, and then change the code in tags.cc to read the bytes and throw them away, rather than reading the bytes and storing them in the player data.